public/js/servers.js

```diff
@@ -30,6 +30,48 @@ document.addEventListener('DOMContentLoaded', () => {
         'YT', 'ZA', 'ZM', 'ZW'
     ];
 
+    function prettyPrintMap(mapId) {
+        const maps = {
+            // Gridmaps
+            gridmap: 'Gridmap',
+            smallgrid: 'Gridmap V2',
+
+            // USA
+            east_coast_usa: 'East Coast USA',
+            west_coast_usa: 'West Coast USA',
+            utah: 'Utah',
+
+            italy: 'Italy',
+
+            // Islands
+            jungle_rock_island: 'Jungle Rock Island',
+            small_island: 'Small Island',
+
+            // Generic
+            industrial: 'Industrial Site',
+            derby: 'Derby Arenas',
+            hirochi_raceway: 'Hirochi Raceway',
+
+            automation_test_track: 'Automation Test Track',
+            driver_training: 'Driver Training',
+        };
+        
+        const prettyMap = maps[mapId];
+        if (prettyMap) {
+            return prettyMap;
+        }
+
+        // return mapId;
```

Comment:
Request to remove hardcoding logic, avoid this in this proj for maintained versatility and modern behaviour: https://github.com/Xevrac/beamng.cluster/pull/5/commits/8ae877e2fd68b0b7294b483c8f6042bfcc68eef2#diff-b215b50bbb32854108b34257783bbc94a99426c6912f67d1ebc6dd3c805b4b42R33-R64

Reply:
@purifiedfr, if you can conditionalize it so it runs a check on a mapNames.json for example and if it finds a match to override the split method that would be a good compromise. Otherwise, I also don't feel bothered if it just says "Derby". 

I'll leave that to you to decide now :) Good discussion

Reply:
Hey @purifiedfr did you have a decision here? Sorry I've ghosted this, I've returned to work full-time which has hampered my free time to work on this project.